forest/daemon/src/daemon.rs

```diff
@@ -538,37 +538,54 @@ async fn prompt_snapshot_or_die(
 async fn sync_from_snapshot<DB>(
     config: &Config,
     state_manager: &Arc<StateManager<DB>>,
+    halt_after_import: bool,
```

Comment:
This is a cosmetic thing, but this makes it hard to understand for the fellow devs. If you are passing `halt_after_import` boolean, I'd expect this method would do the halting itself, which isn't the case (and rightly so).

`sync_from_snapshot(&config, &state_manager, opts.halt_after_import).`

A bit more expressive form would be

`sync_from_snapshot(&config, &state_manager, ProofParametersFetch::EnsureDownloaded).`

what do you think?

Reply:
Updated